site/app/controllers/MiscController.php

```diff
@@ -54,35 +54,24 @@ public function run() {
 
     private function encodePDF(){
         $gradeable_id = $_POST['gradeable_id'] ?? NULL;
-        $user_id = $_POST['user_id'] ?? NULL;
+        $id = $_POST['user_id'] ?? NULL;
         $file_name = $_POST['filename'] ?? NULL;
 
-        $gradeable = $this->core->getQueries()->getGradeableConfig($gradeable_id);
-        if ($gradeable === null) {
-            $this->core->getOutput()->renderJsonError("You do not have access to this file");
-            return false;
-        }
-
-        $submitter = $this->core->getQueries()->getSubmitterById($user_id);
-        if ($submitter === null) {
-            $this->core->getOutput()->renderJsonError("You do not have access to this file");
-            return false;
-        }
-        $graded_gradeable = $this->core->getQueries()->getGradedGradeableForSubmitter($gradeable, $submitter);
-
-        if ($graded_gradeable === null) {
-            $this->core->getOutput()->renderJsonError("You do not have access to this file");
-            return false;
+        $config = $this->core->getQueries()->getGradeableConfig($gradeable_id);
+        if($config->isTeamAssignment()){
```

Comment:
this if/else block is unnecessary.  There should be a 'by-submitter-id' version of this function